src/sass/utilities/_basic-list.scss

```
@@ -1,40 +1,48 @@
-
-        >li:not(:first-child) {
-            margin-top: 0.5em;
-        }
-    }
-
-    &.no-bullets {
-        list-style-type: none;
-        margin-left: -2em;
-    }
+	font-family: var(--ff-serif);
+	// bullets flush to the left
```

Stylelint (17.6.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)